### PR TITLE
librsvg: Fix CREW_OPTIONS breakage

### PR DESCRIPTION
**Pull request by satmandu**
For https://github.com/skycocker/chromebrew/issues/4496

Works properly:
- [ ] x86_64 compiling...

### CODE REVIEW

File: packages/librsvg.rb [hunk 1]
```diff
@@ -25,7 +25,12 @@ def self.build
...
-      #{CREW_OPTIONS},
+      "--prefix=#{CREW_PREFIX}",
+      "--libdir=#{CREW_LIB_PREFIX}",
+      "--mandir=#{CREW_MAN_PREFIX}",
+      "--build=#{CREW_BUILD}",
+      "--host=#{CREW_BUILD}",
+      "--target=#{CREW_BUILD}",
       "--enable-introspection=yes",
       "--enable-vala=yes",
       "--disable-static",
```
Comment (uberhacker) [inline packages/librsvg.rb:36]: Try this instead:
```
system "./configure #{CREW_OPTIONS} --enable-introspection=yes --enable-vala=yes --disable-static"
```

Comment (uberhacker) [reply packages/librsvg.rb:36]: The problem is with the way ruby parses strings separated by commas.  `CREW_OPTIONS` is space delimited.

Comment (satmandu) [reply packages/librsvg.rb:36]: Right, but if you have CREW_OPTIONS  on a different line, then the other lines are ignored, and so is CREW/MESON_OPTIONS.

So all following options have to be space delimited on that line?

Comment (uberhacker) [reply packages/librsvg.rb:36]: Yes, unfortunately the ruby parser gets confused when it builds the command if you mix and match space and comma delimited arguments.

**Pull request merged**